src/gluonts/evaluation/_base.py

```diff
@@ -91,6 +89,7 @@ def __init__(
         self.seasonality = seasonality
         self.alpha = alpha
         self.calculate_owa = calculate_owa
+        self.zero_tol = 1e-8
```

Comment:
Why exactly this number?
Isn't there in numpy something like `epsilon`: https://stackoverflow.com/questions/19141432/python-numpy-machine-epsilon